.circleci/config.yml

```diff
@@ -90,12 +90,15 @@ jobs:
         working_directory: ~/transformers
         docker:
             - image: circleci/python:3.6
-        resource_class: small
+        resource_class: medium
         parallelism: 1
         steps:
             - checkout
-            - run: sudo pip install black
+            - run: sudo pip install --editable .
+            - run: sudo pip install torch tensorflow tensorboardX scikit-learn
+            - run: sudo pip install black git+git://github.com/timothycrosley/isort.git@e63ae06ec7d70b06df9e528357650281a3d3ec22#egg=isort
```

Comment:
shoud we ask them politely for a new release?

Reply:
A couple others asked already. I'm not sure this would make a big difference.